### DISCUSSION TITLE
2004/12/22/%E5%8F%A6%E4%B8%80%E5%BC%A0%E5%9C%A8%E6%95%85%E5%AE%AB%E7%9A%84%E7%85%A7%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#4] [Announcements]
# 2004/12/22/%E5%8F%A6%E4%B8%80%E5%BC%A0%E5%9C%A8%E6%95%85%E5%AE%AB%E7%9A%84%E7%85%A7%E7%89%87

而且 msn space 还是 beta 版，好多功能都不好使。

https://blog.qizhen.xyz/2004/12/22/%E5%8F%A6%E4%B8%80%E5%BC%A0%E5%9C%A8%E6%95%85%E5%AE%AB%E7%9A%84%E7%85%A7%E7%89%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
mftony
2009年11月28日 12:25
年輕人,您將是非常有成就的!

```
